source/fe/mapping_q.cc

```diff
@@ -77,7 +77,7 @@ MappingQ<dim,spacedim>::MappingQ (const unsigned int degree,
   // created via the shared_ptr objects
```

Comment:
While you are here: would you mind using `make_shared` on line 74? I only noticed this because of #5883 :)